src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADDBOOK = 'bookstore/src/redux/books/ADD_BOOK';
+const REMOVEBOOK = 'bookstore/src/redux/books/REMOVE_BOOK';
+
+export const addBook = {
+  type: ADDBOOK,
+  id: 0,
+  title: 'Book 0',
+  author: 'Author 0',
+};
+
+export const removeBook = {
+  type: REMOVEBOOK,
+  id: 0,
+};
+
+const bookReducer = (state = [], action) => {
```

Comment:
- Please, kindly create an initialState variable as required in the project requirement like this

` const initialState = [];`
`const bookReducer = (state = initialState, action) => {`

Reply:
Oki, it is done